zvariant/src/signature.rs

```diff
@@ -361,6 +362,18 @@ impl<'a> TryFrom<&'a str> for Signature<'a> {
     }
 }
 
+/// Try to create a Signature from a `Cow<str>.`
+impl<'a> TryFrom<Cow<'a, str>> for Signature<'a> {
```

Comment:
I think the new impls here deserve their own commits.

Reply:
done